Packs/CommonPlaybooks/Playbooks/playbook-Block_Email_-_Generic.yml

```diff
@@ -41,40 +44,39 @@ tasks:
       type: title
       iscommand: false
       brand: ""
-      description: ''
     separatecontext: false
+    continueonerrortype: ""
     view: |-
       {
         "position": {
-          "x": 450,
-          "y": 440
+          "x": 265,
+          "y": 370
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   "3":
     id: "3"
-    taskid: b30d383c-cdfb-4200-8ed6-cb7ab5b00a1c
+    taskid: 3ba3c401-b5ae-4213-83a4-1d9b0f3de689
     type: playbook
     task:
-      id: b30d383c-cdfb-4200-8ed6-cb7ab5b00a1c
+      id: 3ba3c401-b5ae-4213-83a4-1d9b0f3de689
       version: -1
-      name: Symantec block Email
+      name: Symantec - Block Email
```

Comment:
Since `Block Email - Generic` is from 5.0, changing the name of a subplaybook tasks is not supported, need to revert this